kano/webapp.py

```diff
@@ -94,6 +94,9 @@ def run(self):
         view.connect('close-web-view', self._close)
         view.connect('onload-event', self._onload)
 
+        language = self._get_browser_lang()
+        view.execute_script("window.navigator.userLanguage = '%s'" % language)
```

Comment:
I don't think this actually has any affect since according to the [documentation](https://developer.mozilla.org/en-US/docs/Web/API/NavigatorLanguage/language) for `navigator.language` it's a read only property.

The users language should be setup correctly if the user's language is set in the operating system anyway. What was the issue you were having with it?